share/qtcreator/sfdk/modules/20-building-mb2/doc/module.adoc

```diff
@@ -12,49 +12,113 @@ Start by choosing the build target. List the available build tools.
 
 Each build target lets you build software for a specific operating system version running on a specific hardware. Each target is listed under one tooling supporting the given OS version (see MANAGING SDK BUILD TOOLS below for more information).
 
-As with other configuration options, you can decide if you want to apply your choice globally or just for the current (shell) session. Let's pick the latest ARM target for global use and its i486 counterpart for this session only.
+Let's pick the latest 64-bit ARM target for global use.
```

Comment:
4.2.0.21 is not the latest target, maybe it's better to just drop the word.

Reply:
This is mean to refer to the latest target on user's list of installed targets. Added the words there.

Reply:
Yes, of course. I would have realized that if github review tool had not decided to cut the displayed text at just after the list. But I guess the added words don't harm.